kernelci.org

```diff
@@ -211,34 +211,34 @@ cmd_docker() {
     for clang in clang-11 clang-14 clang-15 clang-16 clang-17; do
         for arch in arm arm64 armv5 mips riscv64 x86; do
 	    ./kci_docker $base_args --push $clang --arch $arch \
-                     --fragment=kselftest --fragment=kernelci
+                     --fragments kselftest kernelci
```

Comment:
Are you sure this works?  I would have expected `--fragments="kselftest kernelci"` to make it clear that this string was the one for `--fragments` since it's not an `nargs` argument.

Reply:
Yes, it works fine locally.
`--fragments` is `nargs` and is receiving list of arguments:
```
 {
            'name': '--fragments',
            'nargs': '*',
            'help': "Extra fragments, e.g. kernelci",
        },
```

Reply:
Ah, then I'm not sure if we should have `--fragments` or just `fragments` and use that as positional arguments.

Reply:
I'd prefer `--fragments` as it is easier to distinguish arguments this way.
We can think of it as user's pov.